commandhelper/commandhelper.go

```diff
@@ -0,0 +1,75 @@
+package commandhelper
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/bitrise-io/go-steputils/output"
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// RawOutputCouldNotBeExporterError ...
+type RawOutputCouldNotBeExporterError struct {
```

Comment:
struct can be the error itself, no need to wrap error type in a new field

Reply:
This error won't be needed as `RunAndExportOutputWithReturningLastNLines` will return both the export and cmd errors.